src/views/AuthView.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+import { computed, onBeforeMount, onMounted } from 'vue'
+import { useRoute, useRouter } from 'vue-router'
+
+import useAuthStore from '@/stores/auth.store'
+import logo from '@/assets/brand/bakano-negro.png'
+import ContainerWrapper from '@/components/layout/ContainerWrapper.vue'
+import SpinnerLoader from '@/components/shared/SpinnerLoader.vue'
+import { SpinnerSize } from '@/enum/components/shared/SpinnerLoader.enum'
```

Comment:
this have to be first in imports cause are between "{}"

remember order according lenght